TestHttpQueryListener is flaky

```
2021-09-29T18:15:24.0077563Z [ERROR] Tests run: 13, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 12.279 s <<< FAILURE! - in TestSuite
2021-09-29T18:15:24.0079316Z [ERROR] io.trino.plugin.httpquery.TestHttpQueryListener.teardown  Time elapsed: 5.077 s  <<< FAILURE!
2021-09-29T18:15:24.0080982Z java.io.IOException: Gave up waiting for executor to shut down
2021-09-29T18:15:24.0082207Z 	at okhttp3.mockwebserver.MockWebServer.shutdown(MockWebServer.java:435)
2021-09-29T18:15:24.0083742Z 	at okhttp3.mockwebserver.MockWebServer.close(MockWebServer.java:883)
2021-09-29T18:15:24.0085673Z 	at io.trino.plugin.httpquery.TestHttpQueryListener.teardown(TestHttpQueryListener.java:205)
2021-09-29T18:15:24.0087804Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-09-29T18:15:24.0089917Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-09-29T18:15:24.0092377Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-09-29T18:15:24.0094214Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2021-09-29T18:15:24.0095892Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2021-09-29T18:15:24.0097968Z 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
2021-09-29T18:15:24.0099672Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
2021-09-29T18:15:24.0101118Z 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
2021-09-29T18:15:24.0102514Z 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2021-09-29T18:15:24.0104026Z 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2021-09-29T18:15:24.0105832Z 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2021-09-29T18:15:24.0107639Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2021-09-29T18:15:24.0109394Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-09-29T18:15:24.0111207Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-09-29T18:15:24.0112429Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2021-09-29T18:15:24.0112940Z 
2021-09-29T18:15:24.0612596Z [INFO] 
2021-09-29T18:15:24.0613884Z [INFO] Results:
2021-09-29T18:15:24.0614894Z [INFO] 
2021-09-29T18:15:24.0615891Z [ERROR] Failures: 
2021-09-29T18:15:24.0619468Z [ERROR]   TestHttpQueryListener.teardown:205 » IO Gave up waiting for executor to shut d...
```

https://github.com/trinodb/trino/runs/3747560759

--- a/plugin/trino-http-event-listener/src/test/java/io/trino/plugin/httpquery/TestHttpQueryListener.java
+++ b/plugin/trino-http-event-listener/src/test/java/io/trino/plugin/httpquery/TestHttpQueryListener.java
@@ -200,9 +200,13 @@ public class TestHttpQueryListener
 
     @AfterMethod(alwaysRun = true)
     public void teardown()
-            throws IOException
     {
-        server.close();
+        try {
+            server.close();
+        }
+        catch (IOException ignored) {
+            // MockWebServer.close() method sometimes throws 'Gave up waiting for executor to shut down'
+        }
     }
 
     /**

